README.md

```diff
@@ -1,38 +1,42 @@
 
 - Official website: [js.devexpress.com](https://js.devexpress.com)
 - Pricing: [js.devexpress.com/buy](https://js.devexpress.com/Buy)
-- Licensing: [js.devexpress.com/licensing](https://js.devexpress.com/Licensing)
+- Licensing FAQ: [js.devexpress.com/licensing](https://js.devexpress.com/Licensing)
 - Support: [www.devexpress.com/support](https://www.devexpress.com/support)
 
-
 ## Getting Started
 
-- DevExtreme Angular
-    - [Create a DevExtreme Angular Application](https://js.devexpress.com/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/)
-    - [Add DevExtreme to an Angular Application](https://js.devexpress.com/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/)
+The DevExtreme website includes "Getting Started" guides for every flavor of DevExtreme:
 
-- DevExtreme React
-    - [Create a DevExtreme React Application](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/)
-    - [Add DevExtreme to a React Application](https://js.devexpress.com/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/)
+- Angular
+    - [Create a DevExtreme Angular Application](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/)
+    - [Add DevExtreme to an Angular Application](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/)
 
-- DevExtreme Vue
-    - [Create a DevExtreme Vue Application](https://js.devexpress.com/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/)
-    - [Add DevExtreme to a Vue Application](https://js.devexpress.com/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/)
+- React
+    - [Create a DevExtreme React Application](https://js.devexpress.com/React/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/)
+    - [Add DevExtreme to a React Application](https://js.devexpress.com/React/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/)
 
-- DevExtreme jQuery
-    - [Add DevExtreme to a jQuery Application](https://js.devexpress.com/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/)
+- Vue
+    - [Create a DevExtreme Vue Application](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/)
+    - [Add DevExtreme to a Vue Application](https://js.devexpress.com/Vue/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/)
+
+- jQuery
+    - [Add DevExtreme to a jQuery Application](https://js.devexpress.com/jQuery/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/)
 
 ## Learn
```

Comment:
I would also add an entry for our new UI Templates: https://js.devexpress.com/Templates/UITemplates/